config/locales/stamp_duty.cy.yml

```diff
@@ -8,8 +8,9 @@ cy:
     pretitle: "Defnyddiwch y gyfrifiannell hon i gyfrifo faint o dreth stamp y bydd angen ichi ei thalu ar gartref newydd"
     h1: "Cyfrifiannell treth stamp"
     title: "Cyfrifwch y dreth stamp ar eich eiddo preswyl"
-    subtitle: "Rhaid i chi dalu Treth Stamp (neu Dreth Dir y Dreth Stamp i roi ei henw'n llawn) os prynwch eiddo yn y Deyrnas Unedig sy'n costio mwy na £125,000. Defnyddiwch y gyfrifiannell hon i gyfrifo faint o Dreth Stamp fydd angen i chi ei thalu os ydych yn prynu cartref newydd."
     second_subtitle: "Cyfrifwch y Dreth Stamp ar eich eiddo preswyl yng Nghymru, Lloegr neu Ogledd Iwerddon. Rhaid i chi dalu Treth Stamp (neu Dreth Dir y Dreth Stamp i roi ei henw’n llawn) os prynwch eiddo sy’n costio mwy na £125,000. Defnyddiwch y gyfrifiannell hon i gyfrifo faint o Dreth Stamp fydd yn daladwy. Os ydych yn byw yn yr Alban, mae Treth Stamp wedi ei ddileu ond efallai y bydd angen i chi dalu Treth Trafodion Tir ac Adeiladau (LBTT)."
+    subtitle: "Cyfrifwch y Dreth Stamp ar eich eiddo preswyl yng Nghymru, Lloegr neu Ogledd Iwer    ddon. Rhaid i chi dalu Treth Stamp (neu Dreth Dir y Dreth Stamp i roi ei henw’n llawn) os prynwch eiddo     sy’n costio mwy na £125,000. Defnyddiwch y gyfrifiannell hon i gyfrifo faint o Dreth Stamp fydd yn daladwy."
```

Comment:
It might be a Welsh thing, but the subtitle on line 12 looks a bit odd. ```Iwerddoon``` ---> ```Iwer   ddon```

There seems to be some suspicious spaces later on ``` eiddo     sy’n````

Again, as far as I know it's a Welsh thing and could be fine.